### PR TITLE
Add Android APK build CI and Release version 0.13.0

### PR DESCRIPTION
**Pull request by enfipy**
This PR adds APK build CI as was said in [this comment](https://github.com/RustAudio/cpal/pull/468#issuecomment-717177006).

Also, publish a new release to update Rodio.

### CODE REVIEW

**Review by mitchmindtree — APPROVED** [review]
Amazing, thanks for this @enfipy! I didn't realise it was already so easy to build APKs with Rust at this point :)

I'm happy to land this and publish 0.13. If anyone has any objections let me know, otherwise I'll merge this in the next day or two :+1: cc @est31, @ishitatsuyuki

**Comment by enfipy** [comment]
@mitchhentges Hey, thanks! Please don't merge it as we found some bug with the newest Android SDK 30 that should be fixed in oboe. I will try to update it as soon as possible.

So for now we can just wait for any other objections.

**Review by est31 — APPROVED** [review]

**Comment by enfipy** [comment]
@mitchhentges Finally, can be merged!

**Comment by enfipy** [comment]
Shameless Ping 🙃

File: Cargo.toml [hunk 1]
```diff
@@ -47,7 +46,27 @@ js-sys = { version = "0.3.35" }
...
+
+[[example]]
+name = "feedback"
+path = "examples/feedback.rs"
+crate-type = ["cdylib"]
+
+[[example]]
+name = "record_wav"
+path = "examples/record_wav.rs"
+crate-type = ["cdylib"]
```
Comment (hugovdm) [inline Cargo.toml:72]: The addition of these [[example]] lines breaks "cargo run --example" for me. If I remove these, it works again.

I've done a "rustup update" and a "cargo update", but I don't know what causes cargo itself to get updated, and I don't know Rust very well:

$ cargo --version
cargo 1.47.0 (f3c7e066a 2020-08-28)

Comment (mitchmindtree) [reply Cargo.toml:72]: Likewise, we also ran into this while testing the examples recently. I wonder if it is possible to conditionally change the crate-type based on the OS? I'm not sure what the right solution is here.

Comment (enfipy) [reply Cargo.toml:72]: Perhaps we need to create a new example directly for android? And remove `crate-type` from other examples

**Pull request merged**